packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1159,13 +1160,51 @@ Array of [User](#user-object)
 Due to an infrastructure limitation, [group administrators](https://help.okta.com/en-us/Content/Topics/Security/administrators-group-admin.htm), [help desk administrators](https://help.okta.com/en-us/Content/Topics/Security/administrators-help-desk-admin.htm),
 and [custom administrators](https://help.okta.com/en-us/Content/Topics/Security/custom-admin-role/custom-admin-roles.htm) who are only scoped to view and manage users of their assigned groups may experience timeout for the list users endpoints.
 
-#### List all Users
 
+#### List Users with search
 
-Returns a list of all users that do not have a status of `DEPROVISIONED`, up to the maximum (200 for most orgs)
+Searches for users based on the properties specified in the search parameter. This method typically offers the best performance of any [List Users](#list-users) operation.
```

Comment:
Is it better than "List All Users"?

Reply:
Certainly.  While you CAN use the "list all users" operation to emulate searching for any particular user, by cross-referencing yourself, the performance will be really terrible compared to using a search.  Right?

Reply:
A bit less facetiously - a) It's not exactly wrong (as I demonstrated above), b) it's got the weasel word 'typically', and c) Ordinarily I'm just as nitpicky about half-truths like this as you, but I feel like this is one of the rare circumstances where 1) everyone knows what it means, and 2) it's _more_ legible in the abbreviated format.

Reply:
It depends what you're trying to do.  If you really just want any old users, and are willing to stop at 200, then List All Users is exactly the tool.  It's also a much simpler concept, and has fewer non-default parameters.  It just feels to me like the more natural way to open the topic, and then follow with the three "selective" list operations.  But I don't feel strongly (other than that it made the diff harder to read).

Mostly I was questioning the "best performance".  If what you want is 200 users with no filtering, is it more efficient to search for, what, `profile.login exists` or some such all-inclusive expression?

[Digression: It's really hard to find words to describe what's going on when all the good words are taken.  E.g. I started to write "all-inclusive query" but query is what the `q` parameter stands for.  I only got away with "filtering" because it's in the negative so it can't be confused with `filter`.]

Reply:
True enough.

And I don't see any other convenient way to express that point other than explicitly call it out as an exception.  (Which I didn't really want to do. but isn't actually that problematic now that I've written it out.)